invoice/src/main/java/org/killbill/billing/invoice/generator/InvoiceWithMetadata.java

```diff
@@ -80,6 +83,73 @@ public boolean apply(final InvoiceItem input) {
         });
     }
 
+    public Set<TrackingRecordId> getTrackingIds() {
```

Comment:
I'm guessing that in `0.22`, we will want to expose these in the `Invoice` object. In the meantime, should we at least pass the information to invoice plugins as a custom/hidden plugin property? That way, it could help folks access that information without having to query the table directly.

Since you have it in `InvoiceWithMetadata`, it seems pretty straightforward to send it to `onSuccessCall` and `onFailureCall`. It's a bit more difficult for the `priorCall` but we could maybe punt that for now.